tests/functional/cylc-combination-scripts/vr_workflow_stop/flow.cylc

```diff
@@ -0,0 +1,8 @@
+[scheduling]
+    initial cycle point = 1500
+    [[graph]]
+        R1 = foo
```

Comment:
Again this doesn't match the `sed -i 's@P1Y@P5Y@' flow.cylc` in `tests/functional/cylc-combination-scripts/07-vr-remote.t`